ts/screens/profile/PrivacyMainScreen.tsx

```diff
@@ -176,9 +184,14 @@ class PrivacyMainScreen extends React.PureComponent<Props> {
                 "profile.main.privacy.removeAccount.description"
               )}
               onPress={() =>
-                this.props.loadUserDataRequest(
-                  UserDataProcessingChoiceEnum.DELETE
-                )
+                !this.props.isEmailValidated
```

Comment:
Since these 2 items do almost the same stuff, you could make a function to handle both:

```typescript
onPress={() => this.handleDownloadOrDeleteItemPress(UserDataProcessingChoiceEnum.DOWNLOAD)}
[...]
private handleDownloadOrDeleteItemPress = (
    choice: UserDataProcessingChoiceEnum
  ): void => {
    !this.props.isEmailValidated
      ? Alert.alert(needValidatedEmailTitle, needValidatedEmailSubtitle)
      : this.props.loadUserDataRequest(choice);
  };
```

Could it be an improvement?